### DISCUSSION TITLE
A new category

### DISCUSSION THREAD

**Discussion by georgekart** [v2fly/domain-list-community#1684] [Ideas]
If I was able to provide a list of all domains that are banned in Russia (and a script to update it), could we perhaps add it as its own category so that all users from Russia could simply one category to gain immediate access to all banned domains? I will include a file as an example.
[domainlist.txt](https://github.com/v2fly/domain-list-community/files/11585193/domainlist.txt)

**Comment by MkQtS** [comment]
Sorry for the late response. It's too huge and hard to maintain for this repo.

For Chinese, there's a similar project/list https://github.com/gfwlist/gfwlist. You may find/setup a similar project for Russian.
